pals/riteby.md

```diff
@@ -0,0 +1,57 @@
+
+# Riteby Hindew
+
+> "I love to make music, and many other abstract concepts too!"
+
+#### Species, Class, Level:
+Faun Bard, Level 1
+
+#### Deity
+
+Sehanine
+
+#### Languages known:
+Common, Elven
+
+#### Attributes
+* Cha: 18 (+4)
+* Con: 10 (+0)
+* Dex: 13 (+1)
+* Int: 16 (+3)
+* Str: 8  (-1)
+* Wis: 10 (+0)
+
+#### Defense scores
+
+#### Other scores
+
+#### Powers
+* Majestic Word
+* Words of Friendship
+* Vicious Mockery (At-will)
+* Guiding Strike (At-will)
+* Blunder (Encounter)
+* Stirring Shout (Daily)
+
+#### Trained Skills
+* Arcana
+* Acrobatics
+* Bluff
+* Diplomacy
+* History
+
+#### Feats
+* Escape Artist
+* Skill Versatility
+* Ritual Caster
+  * Glib Limerick
+  * Silence
+
+#### Equipment
+* Dagger (3)
+* Leather Armor
+* Leather Armour
```

Comment:
One set of leather armor is probably enough. (Unless the other is something more _amourous_? Ahahaha do you get it)